crates/trigger/src/runtime_config/variable_provider.rs

```diff
@@ -32,7 +34,7 @@ impl ConfigProviderOpts {
 
 #[derive(Debug, Default, Deserialize)]
 #[serde(deny_unknown_fields)]
-pub struct EnvConfigProviderOpts {
+pub struct EnvVariableProviderOpts {
```

Comment:
Referring to this: https://github.com/fermyon/spin/pull/1918#issuecomment-1771426654

Reply:
Seems fine.